src/core/client/stream/test/postComment.spec.tsx

```diff
@@ -92,25 +66,26 @@ it("renders comment stream", async () => {
 });
 
 it("post a comment", async () => {
+  // Wait for loading.
+  await timeout();
   testRenderer.root
     .findByProps({ inputId: "comments-postCommentForm-field" })
     .props.onChange({ html: "<strong>Hello world!</strong>" });
 
-  timekeeper.freeze(new Date("2018-07-06T18:24:00.002Z"));
+  timekeeper.freeze(new Date("2018-07-06T18:24:00.000Z"));
+
   testRenderer.root
     .findByProps({ id: "comments-postCommentForm-form" })
     .props.onSubmit();
 
+  timekeeper.reset();
+
   // Test optimistic response.
   expect(testRenderer.toJSON()).toMatchSnapshot();
 
   // Wait for loading.
   await timeout();
 
-  // Travel to the time where the "timeout" has executed.
-  timekeeper.travel(new Date("2018-07-06T18:24:01.002Z"));
-
   // Test after server response.
   expect(testRenderer.toJSON()).toMatchSnapshot();
```

Comment:
When timekeeper is reset above, time begins to tick, so this snapshot (which depends on time) is no longer guaranteed to be deterministic.

Can you explain the changes here?

Reply:
between line `108` and `114`, there is really no logic being executed. So this basically a noop.

Reply:
Except for optimistic responses – which is what the timekeeper covers – we don't have time dependent logic. We avoid it whenever possible, to make testing simpler.

Reply:
Note: snapshot data does not change in a time dependent way.